### PR TITLE
adding redirects so that old website API links surfaced from searches

### PR DESCRIPTION
**Pull request by sad-**
… redirect to proper locations in new website

## Description ##
(Brief description on what this PR is about)

This allows the links in the top results of web searches with queries like "mxnet *** api" redirect to the right location in the new website. working on redirects for search results with links that refer to a specific version as well.

### CODE REVIEW

File: docs/static_site/src/.htaccess [hunk 1]
```diff
@@ -22,3 +22,50 @@ RewriteCond %{REQUEST_URI} !\/$
...
+Redirect 301 /api/python/tools/test_utils.html /api/python/docs/api/advanced/mxnet.test_utils.html
+
+# Top Level Nav bar
+Redirect 301 /install/index.html /get_started
+Redirect 301 /test/get_started/install.html /get_started
+Redirect 301 /faq/index.html /api
+Redirect 301 /tutorials/index.html /api
+Redirect 301 /architecture/index.html /api/architecture/overview
+Redirect 301 /community/ecosystem.html /ecosystem
+Redirect 301 community/powered_by.html /ecosystem
```
Comment (ThomasDelteil) [inline docs/static_site/src/.htaccess]: missing a slash

**Comment by ThomasDelteil** [comment]
@aaronmarkham @marcoabreu, please merge asap if CI passes :+1: this will be very helpful to our users, and if not, please give the CI a little nudge, thanks!

**Comment by ThomasDelteil** [comment]
@aaronmarkham beta doesn't work with current htaccess prod config either, I think it's a bug of beta env, it works on dev http://54.211.76.74

**Comment by aaronmarkham** [comment]
@sad- I thought changing the name of the PR would trigger CI - it has in the past. Anyways, when #16304 gets merged, please rebase this PR, so it'll pass CI.

**Comment by ThomasDelteil** [comment]
closing and reopening under my own fork because current hypothesis is that "sad-" breaks the CI somewhere in bash

**Pull request closed without merging**